### DISCUSSION TITLE
How do you use the Community page?

### DISCUSSION THREAD

**Discussion by rmathew1992** [community/community#40459] [Repositories]
### Select Topic Area

Product Feedback

### Body

👋 We'd love to know how you use the Community page on the Repository Insights tab (navigate to https://github.com/[REPO_NAME]/discussions/graphs/community), or click on "Insights" < "Community" from your repository.

<img width="1613" alt="Screen Shot 2022-11-30 at 10 28 43 AM" src="https://user-images.githubusercontent.com/2595311/204853624-397618f3-eb6b-46e1-a158-88e871d77cab.png">
<img width="1352" alt="Screen Shot 2022-11-30 at 11 26 56 AM" src="https://user-images.githubusercontent.com/2595311/204853626-5a11d494-094c-400d-b388-16f5baac10d7.png">

**Feedback we'd love to hear from you**

* How often do you use these graphs?
* What data are you looking to get from it?
* What actions do you expect to take using the data from these graphs?
* What could we do to make this data easier to use or better for you?
* Anything else you think we should know?

**Comment by monacofj** [comment]
There it goes.

>     * How often do you use these graphs?

A few times a week.

>     * What data are you looking to get from it?

I want to know how active and productive the community is.
In the Contribution Activity graph, if the curve of PRs follows the curve of Issues, I see that things are running smoothly. If Issues goes up and PRs don't follow, that's is not so good. 

>     * What actions do you expect to take using the data from these graphs?

If issues are accumulating, I discuss with developers.

>     * What could we do to make this data easier to use or better for you?

I would like to see

- a time plot of open and closes issues (perhaps a line plot, or a stack-bar plot)
- a plot telling me when an issue was created and when it was closed, so that I can estimate how long it takes for issues to be solved, perhaps with some statistics too, like averages (perhaps a scatter plot with open x close).


>     * Anything else you think we should know?

This page is very important for project managers.

**Comment by msavy** [comment]
Not sure if I've not set my locale correctly somewhere in my GitHub Profile,  but your graphs seem to display only American-style month/day date format, which was quite confusing until I realised what was going on.

**Comment by gwillcox-r7** [comment]
The limit of many of GitHub's insights to a year when sites like https://ossinsight.io/ allow it to span several years with graphs showing changes over time limits the usability of a lot of this functionality. We often want to see how things change over longer time frames, not just within the short term period. Right now the functionality of a lot of these GitHub Insight features is like looking at a snapshot in time, with no reference as to how that might have changed vs the previous snapshot.

**Comment by pransh15** [comment]
Hey @rmathew1992 👋 
Thank you for asking this question, I was looking to give feedback on this to someone but I did not know where to do it. 😅 

How often do you use these graphs?
> Not a lot, because I don't get to see as much data as I'd like. I use it for reporting and understanding engagement, and which external website does traffic come from.

What data are you looking to get from it?
> When I mention data, I would like some more filtering capabilities too, where I can see last week, and a date range, and more.
> - Number of issues
> - Number of PRs
> - Number of Contributors
> - Lurkers, maybe(?)
> - Discussion Page Views
> - Org Page Views - since I have no idea on who landed where apart from the repos.

What actions do you expect to take using the data from these graphs?
> Focus on adding more engagement and content on GitHub to engage users, and add more Q&As here to make it more accessible for our users to find answers to their questions.

What could we do to make this data easier to use or better for you?
> I'd like a way to export the data and understand it wider in context since our OSS product lives here, gets cloned here, and the community contributes here. If I could build a lifecycle on GitHub and make it easier to engage here, the analytics would contribute to that.

Anything else you think we should know?
> Feature Request: Can we have Discussions at the forefront on the org/repo page? It would be more accessible for us to be able to do that, where we can mention announcements and cards, and get our community engaged from the get-go!

**Comment by adegans** [comment]
Where’s the download counter for releases… or something to that extent.

**Comment by markseuffert** [comment]
I would like to see a list of the most helpful community members at the end of the year. 

Active community members are infinitely valuable for our community, they answer questions and are a true time saver for maintainers. A the moment the discussions page shows `Most helpful` of the last 30 days, something similar would be useful on the community page. Perhaps show a list of the most helpful, most active community members? Perhaps give an achievement "gold medalist" to the most helpful of the year?

**Comment by ghost** [comment]
hello check my github page, im making follow for follow :) just follow, i'll turn back

**Comment by SbSh-DNA-ModDing** [comment]
a statistic about download traffic from visitors would be nice too

**Comment by TonnyFexxy** [comment]
> ### Select Topic Area
> Product Feedback
> 
> ### Body
> 👋 We'd love to know how you use the Community page on the Repository Insights tab (navigate to [https://github.com/[REPO_NAME]/discussions/graphs/community](https://github.com/%5BREPO_NAME%5D/discussions/graphs/community)), or click on "Insights" < "Community" from your repository.
> 
> <img alt="Screen Shot 2022-11-30 at 10 28 43 AM" width="1613" src="https://user-images.githubusercontent.com/2595311/204853624-397618f3-eb6b-46e1-a158-88e871d77cab.png">
> 
> <img alt="Screen Shot 2022-11-30 at 11 26 56 AM" width="1352" src="https://user-images.githubusercontent.com/2595311/204853626-5a11d494-094c-400d-b388-16f5baac10d7.png">
> 
> **Feedback we'd love to hear from you**
> 
> * How often do you use these graphs?
> * What data are you looking to get from it?
> * What actions do you expect to take using the data from

**Comment by SaschaMann** [comment]
> Anything else you think we should know?

In a repo where I have write access but not more, it prompts me to "Set up discussions" on the Insights -> Community page (`https://github.com/org/repo/graphs/community`). Clicking on it leads to a 404 page (`https://github.com/org/repo/discussions/welcome_templates`), presumably because I lack the permissions to set up discussions. A different prompt like "ask your admin to enable discussions" would work better, I think.

**Comment by jlord** [comment]
👋 Thanks all! This was great feedback for our planning ✨
